scripts/_types/otel_types.py

```diff
@@ -0,0 +1,39 @@
+from typing import (
+    List,
+    TypedDict,
+)
+
+
+class OTelAttribute(TypedDict, total=False):
+    id: str
+    stability: str
+    ref: str
+    deprecated: str
+    group_display_name: str
+
+
+class OTelGroup(TypedDict, total=False):
+    id: str
+    type: str
+    prefix: str
```

Comment:
prefix is going to be removed in the next release of otel

Reply:
yes, I know, but it's still being used in some places, so added it as a fallback